bin.src/genDefectList.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+import argparse
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+from lsst.log import Log
+import numpy as np
+import os
+import pfs.drp.stella as drpStella
+try:
+    import lsst.afw.display as afwDisplay
+except ImportError:
+    afwDisplay = None
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--arm", help="Arm for which to create the defect list", type=str, default='r', choices=['b', 'n', 'm', 'r'])
+parser.add_argument("--badCols", help="List of bad columns delimited by ','", type=str, default="1023,1024,3071,3072")
+parser.add_argument("--display", help="Set to display outputs", action="store_true")
+parser.add_argument("--debug", help="Set to print debugging outputs", action="store_true")
+parser.add_argument("--expTimeHigh", help="Exposure time for stronger flats", type=int, default=30)
+parser.add_argument("--expTimeLow", help="Exposure time for weaker flats", type=int, default=2)
+parser.add_argument("--gapHigh", help="Number of column where gap between physical devices ends", type=int, default=2052)
+parser.add_argument("--gapLow", help="Number of column where gap between physical devices starts", type=int, default=2045)
+parser.add_argument("--maxSigma", help="Maximum sigma for good pixels", type=float, default=6.)
+parser.add_argument("--medianFlatsOut", help="Median Flats output root (without '.fits'). Leave empty for not writing", type=str, default="")
+parser.add_argument("--nCols", help="Number of columns in images", type=int, default=4096)
+parser.add_argument("--nPixCut", help="Number of rows/columns around the edge of images to ignore", type=int, default=40)
+parser.add_argument("--nRows", help="Number of rows in images", type=int, default=4174)
+parser.add_argument("--outFile", help="Output defect list relative to OBS_PFS_DIR", type=str, default="pfs/defects/2015-12-01/defects.dat")
```

Comment:
It's correct now. Also, I don't want lots of different bad-pixel lists for now to make sure that all files have the same bad-pixel list associated.